### PR TITLE
Add missing error handling

### PR DESCRIPTION
**Pull request by ash2k**
Noticed a missing check, then ran a linter and found more =)

### CODE REVIEW

File: context.go [hunk 1]
```diff
@@ -374,11 +389,11 @@ func (lc *listContent) metadata() map[string]interface{} {
 func castToStrings(slice []interface{}) ([]string, error) {
 	result := make([]string, len(slice))
 	for i, v := range slice {
-		if s, ok := v.(string); !ok {
+		s, ok := v.(string)
```
Comment (wryun) [inline context.go:392]: Not sure why this change? Just something that the linter wanted?

**Review by wryun — APPROVED** [review]
Err, oops - thanks! Missed this PR before.

File: context.go (hunk 1 above)
Comment (ash2k) [reply context.go:392]: Yes, and reads a bit better I think.

Comment (wryun) [reply context.go:392]: eh, I like the scoping of the other one, I think (though it doesn't matter in this case).
